File: pgm/ABC_JavaScript/ABC_164_A.js
"use strict";
function main(input) {
    let line = input.split("\n");
    let arr = line[0].split(" ");
    const S = parseInt(arr[0]);
    const W = parseInt(arr[1]);
    if (S <= W) {
        console.log("unsafe");
    } else {
        console.log("safe");
    }
}
main(require('fs').readFileSync('/dev/stdin', 'utf8'));

File: pgm/ABC_R/ABC_164_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
unsafe <- "unsafe"
safe <- "safe"
one <- 1
two <- 2
blank <- ""
space <- " "
S <- strsplit(arr[one], split = space)[[one]][one] |>
    as.integer()
W <- strsplit(arr[one], split = space)[[one]][two] |>
    as.integer()
if (S <= W) {
    unsafe |>
        cat("\n", sep = blank)
} else {
    safe |>
        cat("\n", sep = blank)
}
q("no")